dashboard/test/lib/test_cpa.rb

```diff
@@ -42,20 +43,34 @@ def test_cpa_experience_before_new_user_lockout
   end
 
   def test_cpa_experience_after_new_user_lockout
-    current_time = DateTime.parse('2023-01-02T00:00:01Z')
+    current_time = DateTime.parse('2023-01-01T00:00:01Z')
     schedule = {
-      Cpa::NEW_USER_LOCKOUT => '2023-01-02T00:00:00Z',
+      Cpa::NEW_USER_LOCKOUT => '2023-01-01T00:00:00Z',
+      Cpa::ALL_USER_LOCKOUT_WARNING => '2023-01-02T00:00:00Z',
       Cpa::ALL_USER_LOCKOUT => '2023-01-03T00:00:00Z'
     }
     stub_dcdo(schedule, nil)
     result = Cpa.cpa_experience(@request, current_time: current_time)
     assert_equal Cpa::NEW_USER_LOCKOUT, result
   end
 
+  def test_cpa_experience_after_all_user_lockout_warning
+    current_time = DateTime.parse('2023-01-02T00:00:01Z')
+    schedule = {
+      Cpa::NEW_USER_LOCKOUT => '2023-01-01T00:00:00Z',
+      Cpa::ALL_USER_LOCKOUT_WARNING => '2023-01-02T00:00:00Z',
+      Cpa::ALL_USER_LOCKOUT => '2023-01-03T00:00:00Z'
```

Comment:
Just a note for all reviewers, I'm not sure what the scheduled dates are, but we all need to make sure the dates are accurate. I'm seeing 2023 and I want to make sure it is not supposed to date in 2024.

Reply:
These are in the tests and they just need to be more or less in an order. This is the proper order. Seems good to have them be a set of dates that is easy to think about and understand quickly.